diff -urN a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2017-11-12 13:33:19.478303578 +0100
+++ b/CMakeLists.txt	2017-11-12 13:34:55.358002464 +0100
@@ -8,7 +8,7 @@
 set (INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
 set (DATADIR "${INSTALL_PREFIX}/share")
 
-set (APP_ID "com.github.jmoerman.go-for-it")
+set (APP_ID "de.manuel_kehl.go-for-it")
 set (APP_SYSTEM_NAME ${APP_ID} CACHE STRING "Name to used for file names. For example the file name of the .desktop file.")
 
 set (PKGDATADIR ${DATADIR}/${APP_SYSTEM_NAME})
